### DISCUSSION TITLE
Is there a tutorial on SB remotes yet?

### DISCUSSION THREAD

**Discussion by cliffsadler** [GerryDazoo/Slinger#490] [Q&A] [answered]
The networking tutorial is great.  I kind of understand the remote thing, except in my case, I could never use the HDMI source, so any remote that I had programmed to the SB500 is associated with the component source.  I am now streaming 1080P through HDMI, and I have other solutions for remote control (Harmony Hub), but would like to understand the remote part of this in case I set up my other SB 500 or Harmony decides to quit their cloud service (rumors are there).
The Cable Box I have is from Spectrum, which there is no entry for the Technicolor box on the hifi-remote site.
Am I correct in assuming that at this moment there is no way to create a remote entry for the HDMI input, or learn anything?  I'm guessing that there is a mapping from the SB command, like Power On ( code 2), and what actually gets transmitted by the SB IR emmiter, based on the remote type.  .BIN file?  I tried Slingplayer desktop, and although it can see the SB on the network (after a retry), and names it correctly, it doesn't ever connect to it or advance in the setup screens when I click on Reconfigure.  Initializing just spins.  That is version 5.0.0.x  Maybe it's not really connecting, but instead reading some file that knows the names of my SBs.
Sorry for the longness of the message.  I don't know what I don't know (yet!)

**Comment by swilsonnc** [comment]
The html remote doesn't care about the source.  When you select a command on the remote it send it to the slingbox which sends it to the IR plug which goes to all the IR transmitters.

On the slingbox side previously you ran slingplayer and setup the type of settop for the source.  Can you run slingplayer v1.5 or so and set it up or at least check it.  That could be the problem if it is a keycode issue.

**Reply by cliffsadler** [reply]
The problem appears to be that the set top box I have is pretty new, and does not exist in the list of sources.  Without the mapping of a key button to a specific IR code, I cannot control this box.  Haven't found any way to learn, or create the needed file(s).  I think I am stuck with using the harmony remote, which does work just fine.

**Reply by swilsonnc** [reply]
which model set top box is it?

**Reply by swilsonnc** [reply]
Also, if there are any technicolor choices in slingplayer 1.5 I would try them because it would surprise me if they changed the key codes for those boxes just for a new box.

**Comment by GerryDazoo** [comment, marked as answer]
https://slingbox.fandom.com/wiki/How_to_create_custom_remote_upgrade
http://www.hifi-remote.com/forums/dload.php?action=category&cat_id=97&sid=fc48acc94d863c8f2b03333911ab079f

**Reply by cliffsadler** [reply]
Thanks, Gerry.  Exactly what I was fishing for.
